app/Resources/views/base.html.twig

```diff
@@ -9,6 +9,7 @@
         <meta charset="UTF-8" />
         <meta name="viewport" content="width=device-width, initial-scale=1"/>
         <title>{% block title %}Symfony Demo application{% endblock %}</title>
+        <link rel="alternate" type="application/rss+xml" href="{{ path('blog_rss') }}">
```

Comment:
To be honest I don't know. It makes sense to only use `text/xml` ... but then, will browsers recognize that as a RSS link?